Gameserver/src/bedwars/java/com/minecrafttas/tasbattle/bedwars/components/InventoryManagement.java

```diff
@@ -71,14 +71,15 @@ public void onTick(ServerTickStartEvent e) {
 		for (var p : this.players) {
 			var inv = p.getInventory();
 			
-			// verify at least one sword in inventory
-			if (!inv.contains(Material.WOODEN_SWORD) && !inv.contains(Material.STONE_SWORD) && !inv.contains(Material.IRON_SWORD) && !inv.contains(Material.DIAMOND_SWORD))
+			// verify the correct sword is in the inventory
```

Comment:
As a sidenote, all comments in this file are missing a "that"
```
// verify that the correct sword is in the inventory
```
Not a dealbreaker here but it sounds weird when reading through them

Reply:
it's OPTIMAL! lol